src/python/scripts/dx-fetch-bundled-depends

```diff
@@ -41,4 +41,4 @@ if 'DX_JOB_ID' in os.environ:
             if dep['id']['$dnanexus_link'].startswith('file-'):
                 print "*** Downloading bundled file", dep['name']
                 dxpy.download_dxfile(dep['id'], dep['name'])
-                subprocess.check_call(['dx-unpack', dep['name']])
+                subprocess.check_call(['dx-unpack', re.escape(dep['name'])])
```

Comment:
I think that's because dx-unpack itself has a bug in its treatment of the filename. We should wait for that fix to land so as not to force callers of dx-unpack to escape the command line.